content/ru/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -0,0 +1,452 @@
+pod/my-nginx-2035384211-u3t6x labeled
+```
+
+Сначала она выберет все Pod'ы с лейблом `app=nginx`, а затем пометит их лейблом `tier=fe`. Чтобы просмотреть список этих Pod'ов, выполните:
+
+```shell
+kubectl get pods -l app=nginx -L tier
+```
+```shell
+NAME                        READY     STATUS    RESTARTS   AGE       TIER
+my-nginx-2035384211-j5fhi   1/1       Running   0          23m       fe
+my-nginx-2035384211-u2c7e   1/1       Running   0          23m       fe
+my-nginx-2035384211-u3t6x   1/1       Running   0          23m       fe
+```
+
+Будут выведены все Pod'ы `app=nginx` с дополнительным столбцом `tier` (задается с помощью `-L` или `--label-columns`).
+
+Дополнительную информацию можно найти в разделах [Лейблы](/docs/concepts/overview/working-with-objects/labels/) и [kubectl label](/docs/reference/generated/kubectl/kubectl-commands/#label).
+
+## Обновление аннотаций
+
+Иногда возникает потребность навесить на ресурсы аннотации. Аннотации — это произвольные неидентифицирующие метаданные для извлечения клиентами API (инструментами, библиотеками и т.п.). Добавить аннотацию можно с помощью команды `kubectl annotate`. Например:
+
+```shell
+kubectl annotate pods my-nginx-v4-9gw19 description='my frontend running nginx'
+kubectl get pods my-nginx-v4-9gw19 -o yaml
+```
+
+```shell
+apiVersion: v1
+kind: pod
+metadata:
+  annotations:
+    description: my frontend running nginx
+...
+```
+
+Для получения дополнительной информации обратитесь к разделу [Аннотации](/docs/concepts/overview/working-with-objects/annotations/) и описанию команды [kubectl annotate](/docs/reference/generated/kubectl/kubectl-commands/#annotate).
+
+## Масштабирование приложения
+
+Для масштабирования приложения можно воспользоваться инструментом `kubectl`. Например, следующая команда уменьшит число реплик с 3 до 1:
+
+```shell
+kubectl scale deployment/my-nginx --replicas=1
+```
+
+```shell
+deployment.apps/my-nginx scaled
+```
+
+После ее применения количество Pod'ов под управлением соответствующего объекта Deployment сократится до одного:
+
+```shell
+kubectl get pods -l app=nginx
+```
+
+```shell
+NAME                        READY     STATUS    RESTARTS   AGE
+my-nginx-2035384211-j5fhi   1/1       Running   0          30m
+```
+
+Чтобы система автоматически выбирала необходимое количество реплик NGINX в диапазоне от 1 до 3, выполните следующую команду:
+
+```shell
+kubectl autoscale deployment/my-nginx --min=1 --max=3
+```
+
+```shell
+horizontalpodautoscaler.autoscaling/my-nginx autoscaled
+```
+
+Теперь количество реплик NGINX будет автоматически увеличиваться и уменьшаться по мере необходимости.
+
+Для получения дополнительной информации см. разделы [kubectl scale](/docs/reference/generated/kubectl/kubectl-commands/#scale), [kubectl autoscale](/docs/reference/generated/kubectl/kubectl-commands/#autoscale) и [horizontal pod autoscaler](/docs/tasks/run-application/horizontal-pod-autoscale/).
+
+## Обновление ресурсов "по месту"
```

Comment:
It's kinda controversial, but I personally think this will sound a bit more "native" in Russian.
```suggestion
## Обновление ресурсов "на месте"
```